content/_index.html

```diff
@@ -34,11 +34,15 @@ <h3>Making services for everyone</h3>
 <div id="separator-image">
   <div class="container">
     <div class="col-xs-12 col-md-4 quote-photo">
-      <img src="/assets/Naomi.jpg" alt="Naomi Kelly">
+      <img src="/assets/Carmen.jpg" alt="Carmen Chu, City Administrator">
     </div>
-    <div class="col-xs-12 col-md-8 naomi-quote">
-      <p class="separator-quote-2">“Whether you’re a single mom looking for affordable housing in the city, a young person helping your parent navigate city services, or a business owner applying for a permit, you should have access to services online that are easy to find and simple to use. Digital Services is a talented team and I’m proud to see their dedication to solving some of toughest challenges alongside our departments.”</p>
-      <p class="separator-quote-3"><b>– Naomi Kelly, San Francisco City Administrator</b></p>
+    <div class="col-xs-12 col-md-8 carmen-quote">
+      <p class="separator-quote-2">San Francisco Digital Services is part of the San Francisco City Administrator's office, led by City Administrator Carmen Chu.
+      </p>
+      <p class="separator-quote-2">Carmen Chu was the elected Assessor for the City and County of San Francisco. Prior to becoming Assessor, Ms. Chu was an elected representative on the San Francisco Board of Supervisors from 2007 to 2013. She served as Deputy Director of the Mayor’s Office of Public Policy and Finance from 2004 to 2007. Ms Chu has over 16 years of governmental management and finance experience with the City and County of San Francisco.</p>
```

Comment:
Typo, missing a period.

Another note is that it feels a bit weird to have Carmen Chu's bio as the 2nd thing on our site. Hopefully we can come up with something else to replace it soon?

```suggestion
      <p class="separator-quote-2">Carmen Chu was the elected Assessor for the City and County of San Francisco. Prior to becoming Assessor, Ms. Chu was an elected representative on the San Francisco Board of Supervisors from 2007 to 2013. She served as Deputy Director of the Mayor’s Office of Public Policy and Finance from 2004 to 2007. Ms. Chu has over 16 years of governmental management and finance experience with the City and County of San Francisco.</p>
```